app/workers/facilities/pssg_download.rb

```diff
@@ -37,8 +36,8 @@ def create_and_save_drive_time_data(drive_time_data)
     end
 
     def extract_id(attributes)
-      name = attributes&.dig('Name')
-      name.partition(':')&.first&.strip!
+      name = attributes&.dig('Sta_No')
+      name&.strip
```

Comment:
I think you could put the `&.strip` at the end of the previous line, and I think we don't need to encapsulate this logic in a method anymore